### PR TITLE
Add .NET MAUI

### PR DESCRIPTION
**Pull request by mattjohnsonpint**
First release of the `sentry.dotnet.maui` SDK

(Wait to merge this until we publish it to Nuget)

### CODE REVIEW

**Comment by mattjohnsonpint** [comment]
@bruno-garcia

File: packages/nuget/Sentry.Maui/3.18.0-preview.1.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Sentry.Maui",
+  "canonical": "nuget:Sentry.Maui",
+  "version": "3.18.0-preview.1",
+  "package_url": "https://www.nuget.org/packages/Sentry.Maui",
+  "repo_url": "https://github.com/getsentry/sentry-dotnet",
+  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/maui"
```
Comment (bruno-garcia) [inline packages/nuget/Sentry.Maui/3.18.0-preview.1.json:7]: Docs will land as 'guides' e.g: https://docs.sentry.io/platforms/dotnet/guides/aspnet/

```suggestion
  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/guides/maui/"
```

**Review by bruno-garcia — DISMISSED** [review]
1 note, other than that :shipit: 

Worth noting craft doesn't push preview releases so this won't get updated until GA of the package

File: packages/nuget/Sentry.Maui/3.18.0-preview.1.json (hunk 1 above)
Comment (mattjohnsonpint) [reply packages/nuget/Sentry.Maui/3.18.0-preview.1.json:7]: There's a redirect, and our other packages use it, so I was copying the pattern - but i'm fine either way.

**Pull request merged**